From: Antonio Alvarez Feijoo <antonio.feijoo@suse.com>
Date: Wed, 2 Sep 2026 15:47:36 +0200
Subject: test(NFS): fix ETIMEDOUT error in rpc.nfsd call

nfsdcld must be running before rpc.nfsd:

```
+ rpc.nfsd
[    6.872734][  T614] NFSD: Unable to initialize client recovery tracking! (-110)
[    6.874020][  T614] NFSD: Is nfsdcld running? If not, enable CONFIG_NFSD_LEGACY_CLIENT_TRACKING.
[    6.875426][  T614] NFSD: starting 90-second grace period (net effffff9)
```

Origin: upstream, https://github.com/dracut-ng/dracut/commit/58b8321fcd85007e66d7404613793428c3efdf93
---
 test/TEST-60-NFS/server-init.sh | 2 ++
 test/TEST-60-NFS/test.sh        | 2 +-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/test/TEST-60-NFS/server-init.sh b/test/TEST-60-NFS/server-init.sh
index 337217f..6218449 100755
--- a/test/TEST-60-NFS/server-init.sh
+++ b/test/TEST-60-NFS/server-init.sh
@@ -95,6 +95,8 @@ mount -t nfsd nfsd /proc/fs/nfsd
 : > /dev/watchdog
 exportfs -r
 : > /dev/watchdog
+nfsdcld
+: > /dev/watchdog
 rpc.nfsd
 : > /dev/watchdog
 rpc.mountd
diff --git a/test/TEST-60-NFS/test.sh b/test/TEST-60-NFS/test.sh
index 7cb7da5..f2fd0ef 100755
--- a/test/TEST-60-NFS/test.sh
+++ b/test/TEST-60-NFS/test.sh
@@ -229,7 +229,7 @@ make_server_rootfs() {
         --add-confdir test-root \
         -a "bash $USE_NETWORK nfs" \
         --add-drivers "nfsd sunrpc lockd" \
-        -I "exportfs pidof rpc.nfsd rpc.mountd dnsmasq tcpdump" \
+        -I "exportfs pidof rpc.nfsd rpc.mountd dnsmasq tcpdump nfsdcld" \
         --install-optional "/etc/netconfig /etc/nsswitch.conf /etc/rpc /etc/protocols /etc/services /usr/etc/nsswitch.conf /usr/etc/rpc /usr/etc/protocols /usr/etc/services rpc.idmapd /etc/idmapd.conf" \
         -f "$TESTDIR"/initramfs.root
 
